Content.Client/SurveillanceCamera/UI/SurveillanceCameraMonitorWindow.xaml

```diff
@@ -1,25 +1,79 @@
 <DefaultWindow xmlns="https://spacestation14.io"
                xmlns:viewport="clr-namespace:Content.Client.Viewport"
+               xmlns:local="clr-namespace:Content.Client.SurveillanceCamera.UI"
                Title="{Loc 'surveillance-camera-monitor-ui-window'}">
-    <BoxContainer Orientation="Horizontal">
-        <BoxContainer Orientation="Vertical" MinWidth="350" VerticalExpand="True">
-            <!-- lazy -->
-            <OptionButton Name="SubnetSelector" />
-            <Button Name="SubnetRefreshButton" Text="{Loc 'surveillance-camera-monitor-ui-refresh-subnets'}" />
-            <ScrollContainer VerticalExpand="True">
-                <ItemList Name="SubnetList" />
-            </ScrollContainer>
-            <Button Name="CameraRefreshButton" Text="{Loc 'surveillance-camera-monitor-ui-refresh-cameras'}" />
-            <Button Name="CameraDisconnectButton" Text="{Loc 'surveillance-camera-monitor-ui-disconnect'}" />
-            <Label Name="CameraStatus" />
+    <BoxContainer Orientation="Horizontal" HorizontalExpand="True" VerticalExpand="True">
+        <!-- Panel with tabs -->
+        <BoxContainer Orientation="Vertical" MinWidth="350" HorizontalExpand="False" VerticalExpand="True">
+            <TabContainer Name="ViewModeTabs" VerticalExpand="True" HorizontalExpand="True">
+                <!-- Camera list tab -->
+                <BoxContainer Name="{Loc 'surveillance-camera-monitor-ui-tab-list'}" Orientation="Vertical" HorizontalExpand="True">
+                    <OptionButton Name="SubnetSelector" HorizontalExpand="True"/>
+                    <Button Name="SubnetRefreshButton" Text="{Loc 'surveillance-camera-monitor-ui-refresh-subnets'}" HorizontalExpand="True"/>
+                    <ScrollContainer VerticalExpand="True" HorizontalExpand="True">
+                        <ItemList Name="SubnetList" HorizontalExpand="True"/>
+                    </ScrollContainer>
+                    <Button Name="CameraRefreshButton" Text="{Loc 'surveillance-camera-monitor-ui-refresh-cameras'}" HorizontalExpand="True"/>
+                </BoxContainer>
+
+                <!-- Map view tab -->
+                <BoxContainer Name="{Loc 'surveillance-camera-monitor-ui-tab-map'}" Orientation="Vertical" VerticalExpand="True" HorizontalExpand="True">
+                    <local:SurveillanceCameraNavMapControl Name="CameraMap"
+                                        VerticalExpand="True"
+                                        HorizontalExpand="True"
+                                        MinSize="350 350"/>
+
+                    <!-- Map legend -->
+                    <BoxContainer Orientation="Horizontal" Margin="0 10 0 10" HorizontalExpand="True" VerticalAlignment="Center">
```

Comment:
Don't need the vertical alignment, the expand doesn't do anything, horizontal orientation is the default, and can use the short form for the margin:
```suggestion
                    <BoxContainer Margin="0 10">
```